index.js

```diff
@@ -1 +1,103 @@
-// Write the helper here :)
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['dustjs-linkedin'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('dustjs-linkedin'));
+  } else {
+    factory(root.dust);
+  }
+}(this, function(dust) {
+
+  dust.helpers = dust.helpers || {};
+
+  dust.helpers.component = function component(chunk, context, bodies, params) {
+
+    // The component name is required to load and render the template.
+    if (!params.name) {
+      return chunk;
+    }
+
+    bodies = bodies || {};
+    params = params || {};
+
+    // Keep a clear reference to the original context.
+    var originalContext = context;
+
+    // By default, components are completely isolated from the outside template
+    // context. So we start with a new, empty context.
+    var componentContext = dust.makeBase(context.global);
```

Comment:
This is copied from what Dust does internally for partials. Question is, is this global property always empty? If not, should it be used here?